GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEditor;
+using UnityEngine;
+using UnityEditor.SceneManagement;
+using UnityEngine.UI;
+using System.IO;
+
+namespace GooglePlayInstant.Editor
+{
+    public class PlayInstantLoadingScreenGenerator
+    {
+        public static void GenerateLoadingScreenScene(string pathToLoadingScreenImage, string assetBundleUrl)
+        {
+            if (!File.Exists(pathToLoadingScreenImage))
+            {
+                Debug.LogError("Loading screen image file cannot be found.");
+            }
+            else if (EditorSceneManager.GetSceneByName("instant-play-loading-screen-scene").IsValid())
+            {
+                Debug.LogError("A generated loading scene already exists.");
```

Comment:
Instead of being an error, maybe it should replace the existing one?

Reply:
Resolved.